lib/cli/output/show.js

```diff
@@ -0,0 +1,32 @@
+const util = require('util');
+const { Spinner } = require('cli-spinner');
+
+const spinner = new Spinner('Fetching request results, please wait...');
+const startSpinner = () => {
+  spinner.setSpinnerString(0);
+  spinner.start();
+};
+
+const stopSpinner = () => {
+  spinner.stop();
+};
+
+const sendOutput = (event) => {
```

Comment:
Starting with this PR, perhaps we can move away from arrow functions to proper functions where there is a bulky function. We had this discussion during the refactoring of tests. The added benefit the the following flow in a module.
```javascript
module.exports...
//functions arranged to follow top-down rule
```